packages/repository/src/mixins/repository.mixin.ts

```diff
@@ -141,6 +141,16 @@ export function RepositoryMixin<T extends MixinTarget<Application>>(
       }
     }
 
+    /**
+     * Register a model class
+     * @param modelClass - Model class
+     */
+    model<M extends Class<unknown>>(modelClass: M) {
+      return this.bind<M>(`models.${modelClass.name}`)
+        .to(modelClass)
+        .tag('model');
```

Comment:
This is definitely an improvement :+1:

However, it still does not allow components (extensions) to create a model binding that can be added to `this.bindings` exported by the component. Have you considered adding also `createModelClassBinding` as I proposed in my earlier comment?

Reply:
Added.